src/richText.js

```diff
@@ -741,36 +739,12 @@ define([
                 placement: 'bottom',
                 title: getTitle,
                 html: true,
-                content: easy_reference_popover({
-                    text: labelText.text(),
-                    ufid: isFormRef ? labelMug.ufid : "",
-                }),
+                content: '<p>' + labelText.text() + '</p>',
                 template: '<div contenteditable="false" class="popover rich-text-popover">' +
                     '<div class="popover-inner">' +
                     '<div class="popover-title"></div>' +
                     (isFormRef ? '<div class="popover-content"><p></p></div>' : '') +
                     '</div></div>',
-                delay: {
-                    show: 0,
-                    hide: 200,
-                },
-            }).on('shown.bs.popover', function() {
-                var type = isFormRef ? 'form' : 'case';
-                analytics.fbUsage("Hovered over easy " + type + " reference");
-                analytics.workflow("Hovered over easy reference");
-                if (isDate || $this.attr("data-date-format")) {
-                    var pos = $(this).offset(),
-                        x = pos.left,
-                        y = pos.top + $(this).height();
-                    $("#" + dateFormatID).click(function () {
-                        $imgs.popover('hide');
-                        dateformats.showMenu(x, y, function (format) {
```

Comment:
Was it intentional that you also killed the date format changer? I think this also depends on the [Delay popover closing](https://github.com/dimagi/Vellum/pull/828/files#diff-0c6adbb645e08e26dfcc96c660406ee4) code.